AntennaTracker/defines.h

```diff
@@ -24,6 +24,12 @@ enum ServoType {
     SERVO_TYPE_CR=2
 };
 
+enum AltSource{
+	ALT_SOURCE_BARO=0,
+	ALT_SOURCE_GPS=1
+
```

Comment:
Minor formatting issue - could remove this extra line from the enum.  Also add a space before the "{"